website/src/pages/api/admin/stats.ts

```diff
@@ -0,0 +1,19 @@
+import { withRole } from "src/lib/auth";
+
+/**
+ * Returns the messages recorded by the backend for a user.
+ */
+const handler = withRole("admin", async (req, res) => {
+  const statsRes = await fetch(`${process.env.FASTAPI_URL}/api/v1/stats/`, {
+    method: "GET",
+    headers: {
+      "X-API-Key": process.env.FASTAPI_KEY,
+    },
+  });
```

Comment:
Can you move this to `oasst_api_file`? Also, unify these 3 endpoints into 1. Cause these endpoints are not being used anywhere except the status page, if they are being used they may have different params

Reply:
Thanks, @notmd for the feedback, I made the requested changes